js/client_exporter/xlsx/xlsx_fill_helper.js

```diff
@@ -13,6 +13,30 @@ const xlsxFillHelper = {
         }
         return result;
     },
+    tryCreateFillFromSimpleFormat: function({ backgroundColor, patternStyle, patternColor } = {}) {
+        if(typeUtils.isDefined(backgroundColor) && !(typeUtils.isDefined(patternStyle) && typeUtils.isDefined(patternColor))) {
```

Comment:
I think we should apply `backgroundColor` if the `patternColor` is defined but `patternStyle` is not defined

Reply:
It is this code:

            onXlsxCellPrepared: e => {
                e.xlsxCell.style.backgroundColor = 'FFFFFF00';
                e.xlsxCell.style.patternColor = 'FF00FF00';
            },

Most likely it will take short time to investigate 'why my fill was not applied?' and then developer will either 'remove unnecessary patternColor' or 'add lost patternStyle' or 'add lost patternStyle and remove backgroundColor'.
This configuration looks like a typo and we should not support it out of the box.

Reply:
It is this code:

            onXlsxCellPrepared: e => {
                e.xlsxCell.style.backgroundColor = 'FFFFFF00';
                e.xlsxCell.style.patternColor = 'FF00FF00';
            },

Most likely, in OOXML it is this:
                        e.xlsxCell.style.fill = {
                            patternFill: {
                                patternType: undefined,
                                foregroundColor: { rgb: 'FF00FF00', },
                                backgroundColor: { rgb: 'FFFF0000', },
                            }
                        };

Excel 2013 doesn't change cell background for both configurations.